en_ewt-ud-dev.conllu

```diff
@@ -2598,7 +2598,7 @@
 3	of	of	ADP	IN	_	5	case	5:case	_
 4	the	the	DET	DT	Definite=Def|PronType=Art	5	det	5:det	_
 5	lab	lab	NOUN	NN	Number=Sing	2	nmod	2:nmod:of	_
-6	were	where	ADV	WRB	Mood=Ind|Tense=Past|Typo=Yes|VerbForm=Fin	10	advmod	10:advmod	_
+6	were	where	SCONJ	WRB	Mood=Ind|Tense=Past|Typo=Yes|VerbForm=Fin	10	mark	10:mark	_
```

Comment:
feats should be `PronType=Rel`? maybe not fixed when spelling was corrected

Reply:
Why PronType=Rel? This is an adverbial clause modifying the verb 'arrested'